source/includes/examples/tf-example-auth-create-custom-role-stagingprod.rst

```diff
@@ -0,0 +1,29 @@
+.. code-block:: 
+   :copyable: true 
+
+   resource "mongodbatlas_custom_db_role" "create_role" {
+     project_id = var.project_id
+     role_name  = "my_custom_role"
+
+     actions {
+       action = "UPDATE"
+       resources {
+         collection_name = ""
```

Comment:
if you want to have the custom role for all collections, no need to say empty collection_name, you can remove the full line:
```
collection_name = ""
```

as that is the default value, and it's more clear not to specify an empty collection_name.    

same for all examples